------------------------------------------------------------------------
r1353 | lathiat | 2006-12-29 13:50:03 +0100 (Fri, 29 Dec 2006) | 2 lines

 * Make sure we return -1 if we fall of the end of consume_labels()

------------------------------------------------------------------------
Index: avahi-core/dns.c
===================================================================
--- avahi-core/dns.c	(revision 1352)
+++ avahi-core/dns.c	(revision 1353)
@@ -400,6 +400,8 @@
         } else
             return -1;
     }
+
+    return -1;
 }
 
 int avahi_dns_packet_consume_name(AvahiDnsPacket *p, char *ret_name, size_t l) {
